serverless/functions/quickstart.mdx

```diff
@@ -13,11 +13,11 @@ categories:
   - serverless
 ---
 
-The Scaleway [Elements Functions platform](https://www.scaleway.com/en/serverless-functions/) makes your functions available, executes them on demand and manages resource allocation for you. A function is a lightweight unit of logic, defining a procedure for changing one element into another. The function remains static, while the variables that pass through it can vary. We show you how to create a functions namespace, create your function, and delete when no longer needed.
+The Scaleway [Elements Functions platform](https://www.scaleway.com/en/serverless-functions/) makes your functions available, executes them on demand and manages resource allocation for you. A function is a lightweight unit of logic, defining a procedure for changing one element into another. The function remains static, while the variables that pass through it can vary. We show you how to create a functions namespace, create your function, and delete it when you no longer need it.
```

Comment:
```suggestion
The Scaleway [Serverless Functions platform](https://www.scaleway.com/en/serverless-functions/) makes your functions available, executes them on demand, and manages resource allocation for you. A function is a lightweight unit of logic, that defines a procedure for changing one element into another. The function remains static, while the variables that pass through it can vary. We show you how to create a functions namespace, create your function, and delete it when you no longer need it.
```

Tu peux consulter le Wording Handbook pour le naming officiel des produits: https://docs.google.com/document/d/1RvTZ0Wv2H4ZJ_CBmXAzmUICc_DWGwF_WyNR8eMF3S7o/edit

+ je pense qu'on utilise plus le mot "Elements", dans le doute on ne mets pas ;) hehe